metaforce.gemspec

```diff
@@ -31,7 +31,7 @@ EOL
   s.add_dependency 'faraday'
 
   s.add_development_dependency 'rake'
-  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rspec', '~> 2.0'
```

Comment:
Bundler installed 2.99 for me.  There were a number of deprecation warnings, but all the specs pass.